### PR TITLE
Fixes #19915: Improve rust build times

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19915

### CODE REVIEW

**Comment by amousset** [comment]
`debug = 0` seems to make a ~10% difference locally, but could be more on slow io.

File: rust.makefile [hunk 1]
```diff
@@ -21,6 +21,9 @@ else ifneq ($(REDHATOS),)
 PKG_INSTALLER := $(YUM)
 endif
 
+# https://matklad.github.io/2021/09/04/fast-rust-builds.html#ci-workflow
+export CARGO_INCREMENTAL = 0
```
Comment (peckpeck) [inline rust.makefile:25]: doesn't this slow down local developper build ?

Comment (amousset) [reply rust.makefile:25]: I never use this makefile for local dev, but it seems to be used for rudder-lang testing so we need to do it differently indeed. I'll update the PR to pass the parameter through jenkins job directly.

**Comment by amousset** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**